core/src/test/java/com/linecorp/armeria/server/ServerBuilderTest.java

```diff
@@ -167,4 +200,26 @@ public void buildIllegalExceptionTest() {
                 .virtualHost("*.example.com").and();
         assertThatThrownBy(sb2::build).isInstanceOf(IllegalStateException.class);
     }
+
+    /**
+     * Makes sure that {@link ServerBuilder#decorator(DecoratingServiceFunction)} works at every service and
+     * virtual hosts and {@link VirtualHostBuilder#decorator(DecoratingServiceFunction)} works only at
+     * its own services.
+     */
+    @Test
+    public void decoratorTest() throws Exception {
+        final HttpClient c = HttpClient.of(server.uri("/"));
```

Comment:
Although this is test code, `c` is not a familiar abbreviation to me.
How about using `client` or `armeriaClient`?